04/app.js

```diff
@@ -13,52 +13,44 @@ if(navElement) {
 const ulElement = document.createElement ('ul');
 navElement.appendChild(ulElement);
 
-for (let i = 0; i < 3; i++) {
+for (let i = 0; i <3; i++) {
 
     const liElement = document.createElement('li');
     ulElement.appendChild(liElement);
 
+    if (liElement) {
     const aElement = document.createElement('a');
     liElement.appendChild(aElement);
-
-    if i === 0;
-    aElement.setAttribute ('href' , '/')
-    aElement.innerText = 'start';
-
-    if i === 1;
-    aElement.setAttribute ('href' , 'gallery')
-    aElement.innerText = 'galeria';
-
-    if i === 2;
-    aElement.setAttribute ('href' , 'contact')
-    aElement.innerText = 'kontakt';
+    
+    const aList = document.querySelectorAll('a');
+    const text = ['start', 'galeria', 'kontakt'];
+    const url = ['/', 'gallery', '/contact'];
```

Comment:
Zamiast tworzyć dodatkową strukturę postaraj się wykorzystać istniejącą tj. `menuItems.forEach(funciton(item) { /* ... */}`.
Gdzie item to kolejny element tablicy czyli obiekt np. `{text: 'start', url: '/'}` więc wystarczy napisać `item.text`, aby pobrać nazwę linku